fabfile/__init__.py

```diff
@@ -68,10 +80,15 @@ def format():
     else:
         for pyfile in py_diff_list:
             FormatFile(pyfile, in_place=True)
-    print ui.info('Formatting complete')
 
-    if len(local('git status --porcelain', capture=True)) > 0:
-        local('git commit -am "Ran autoformatter"')
+
+@task
+def format_js(source):
+    '''Formats the javascript files in source using prettier. The argument
+       source is a string where each file path is separated by a space.'''
+    if len(source) > 0:
+        options = '--trailing-comma all'
```

Comment:
There's also an option for JSX to put the last `>` on the line of the last prop, let's use that too.